src/shared/components/entry/SequenceView.tsx

```diff
@@ -41,6 +41,7 @@ import {
 import { Namespace } from '../../types/namespaces';
 
 import styles from './styles/sequence-view.module.css';
+import LazyComponent from '../LazyComponent';
```

Comment:
Import should go above style imports

Reply:
Right, I missed that one, it is fixed now